src/aks/serviceaccounts.tf

```diff
@@ -0,0 +1,33 @@
+resource "kubernetes_service_account" "azure_devops" {
+  metadata {
+    name = "azure-devops"
+  }
+}
+
+resource "kubernetes_cluster_role" "deployer" {
+  metadata {
+    name = "cluster-deployer"
+  }
+
+  rule {
+    api_groups = [""]
+    resources = ["deployments", "service"]
+    verbs = ["get", "list", "watch", "create", "update", "patch", "delete"]
+  }
+}
+
+resource "kubernetes_cluster_role_binding" "deployer_binding" {
+  metadata {
+    name = "deployer-binding"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "cluster-deployer"
+  }
+  subject {
+    kind      = "ServiceAccount"
+    name      = "azure-devops"
+    namespace = "default"
```

Comment:
```suggestion
```
bind to all namespaces or we need to setup our namespaces bpd, rtd, fa

Reply:
I changed idea, maybe it's better to set each namespace to avoid to grant any access to kube-system namespace

Reply:
That's on the role not on the service account. I will change from cluster role to a normal one